test/functional/spec-runner/index.js

```diff
@@ -180,6 +159,31 @@ function generateTopologyTests(testSuites, testContext, filter) {
   });
 }
 
+function shouldRunSpecTest(requires, spec, filter) {
+  if (requires.authEnabled && process.env.AUTH !== 'auth') {
+    // TODO: We do not have a way to determine if auth is enabled in our mocha metadata
```

Comment:
Do you think it would be worthwhile to file a ticket so that we are keeping track of this TODO?

Reply:
NODE-3488